src/services/job_service.py

```diff
@@ -229,8 +231,9 @@ def status_job(job_uid: UUID) -> Tuple[JobStatus, str, str]:
             message="The job handler does not support the operation",
             debug="The job handler does not implement the 'progress' method",
         )
-    job.status = status
-    job.log = log
+    if job.status != status:
```

Comment:
Think it's better to do a diff on the last log line here.
This will swallow logs from other handlers that behave better.

F.eks:
`if job.log[::-1].split("\n",1)[0][::-1] != log:`